user_auth/auth_backend.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import ModelBackend
+
+
+class EmailBackend(ModelBackend):
+    def authenticate(self, request, **kwargs):
+        UserModel = get_user_model()
+        try:
+            email = kwargs.get("email", None)
```

Comment:
do you need the ", None)" ?
isnt it by defaut

Reply:
also below

Reply:
Yep, I don't need them, it turned out that the get method has two overload methods one with default equals to None so it takes one parameter.